modules/perception/camera/app/obstacle_camera_perception.h

```diff
@@ -61,10 +61,11 @@ class ObstacleCameraPerception : public BaseCameraPerception {
                               const base::BaseCameraModelPtr model,
                               const app::PerceptionParam &perception_param);
   void SetCameraHeightAndPitch(
-      const std::map<std::string, float> &name_camera_ground_height_map,
-      const std::map<std::string, float> &name_camera_pitch_angle_diff_map,
-      const float &pitch_angle_calibrator_working_sensor);
-  bool GetCalibrationService(BaseCalibrationService **calibration_service);
+        const std::map<std::string, float> &name_camera_ground_height_map,
```

Comment:
If the key order is not necessary, use std::unordered_map to get better performance.

Reply:
I prefer to keep this for now, because a lot of other codes are using the same structure. This should not be a big issue as they are just used once in initialization.